backlogger.py

```diff
@@ -170,16 +170,18 @@ def render_influxdb(data):
 
             start = datetime.strptime(issue["created_on"], "%Y-%m-%dT%H:%M:%SZ")
             end = datetime.strptime(issue["updated_on"], "%Y-%m-%dT%H:%M:%SZ")
-            result[status]["leadTime"].append((end - start).total_seconds() / 3600)
+            result[status]["leadTime"].append((end - start).total_seconds())
             if status == "Resolved":
                 result[status]["cycleTime"].append(cycle_time(issue, status_ids))
         for status in status_names:
             count = len(result[status]["leadTime"])
             if status == "Resolved":
                 measure = "leadTime"
-                extra = " leadTime={leadTime} cycleTime={cycleTime}".format(
-                    leadTime=mean(result[status]["leadTime"]),
-                    cycleTime=mean(result[status]["cycleTime"]),
+                extra = " leadTime={leadTime} cycleTime={cycleTime} leadTimeSum={leadTimeSum} cycleTimeSum={cycleTimeSum}".format(
+                    leadTime=mean(result[status]["leadTime"]) / 3600,
```

Comment:
The pattern could also be extracted into a function (e.g. `mean_time_in_hours`).

Reply:
Ok. But I would do that later. For now I think there will be a conflict with #16 so I don't want to make further changes.